bftengine/src/communication/PlainTcpCommunication.cpp

```diff
@@ -159,20 +177,20 @@ class AsyncTcpConnection :
       socket.close();
     } catch (std::exception &e) {
       LOG_ERROR(_logger, "exception, node " << _selfId << ", dest: " << _destId
-                                   << ", connected: " << connected
-                                   << ", ex: " << e.what());
+                                            << ", connected: " << connected
+                                            << ", ex: " << e.what());
     }
 
     LOG_TRACE(_logger, "exit, node " << _selfId << ", dest: " << _destId
-                            << ", connected: " << connected << ", closed: "
-                            << _closed);
+                                     << ", connected: " << connected << ", closed: "
```

Comment:
This re-indentation doesn't seem inherently better than what was here before. I could see wanting to line everything up "connected: connected" over "closed:closed", but this sort of reindentation for reindentations sake is often a source of unnecessary merge conflicts later.

Reply:
I like the update made in 083f284. The restructuring to pair label with value is good. Re-approved. 👍